schema.cc

```diff
@@ -432,6 +432,21 @@ schema::schema(const schema& o)
     init_view_info_from_other(o);
 }
 
+schema::schema(const schema& s, reversed_tag)
+    : _raw(s._raw)
+    , _offsets(s._offsets)
+{
+    _raw._version = utils::UUID(s._raw._version.get_most_significant_bits(), ~s._raw._version.get_least_significant_bits());
```

Comment:
We would, but why would that be the case?

Reply:
I can also invert another sequence of bits if you have a better suggestion, I did this because this seemed easy enough (MSB has a bunch of stuff encoded in it that I was afraid I would break accidentally).

Reply:
Wait. Didn't I say something completely wrong?
I misunderstood the meaning of "bit-flipped" when looking at the commit message and I didn't really look closely enough at the code.
Obviously when you flip even a single bit you'll get a different UUID...
So nevermind.

Reply:
LSB contains bits which determine the type of UUID, we need to preserve these.

Perhaps it's safer to invert the time part (MSB).

Reply:
I thought the MSB has the identity bits:
https://github.com/scylladb/scylla/blob/cb190481863cdd4d90e8aae1e1b0f976a4a7f9c3/utils/UUID.hh#L59-L61

Reply:
There's version in MSB, and variant in LSB.

Reply:
I'll fish out one of those 8 bit sequences that stores no metadata and flip that.